src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  goods: string[]
+};
+
+export const GoodsList: React.FC<Props> = ({ goods }) => {
+  return (
+    <ul>
+      {goods.map(
+        (good) => (
+          <li data-cy="Good" key={good}>{good}</li>
+        ),
+      )}
```

Comment:
```suggestion
      {goods.map(good => (
          <li data-cy="Good" key={good}>
            {good}
          </li>
      ))}
```
What do U think? I find it more readable